style.css

```diff
@@ -84,4 +85,70 @@ form input {
         width: 1em;
     }
 
+}
+
+/* side nav bar */
+
+.earth-icon {
+    left: 0.2em;
+    bottom: 0.5em;
+}
+
+.info-icon {
+    right: 0.5em;
+    bottom: 2.7em;
+}
+
+.star-icon {
+    left: 0;
+    bottom: 0.7em;
+}
```

Comment:
I think this can be simplified some more.

There are 2  problems here:

1. There's no need for 3 classes, it should be maximum 2 because we have just 2 situations: **icon at the beginning** and **icon at the end**.

2. You have custom values, like `0.7`, `2.7`, etc. These are fishy, and should raise an alarm flag. We should instead try and position using percentage values.

Try and rewrite the classes so that the HTML is like this. No `earth-icon` or `star-icon`. Just `icon-link` which is the base class for the sidebar links. Then 2 more `icon-link--left` and `icon-link--right` which just change the icon position.

```
<a class="icon-link icon-link--left">
  ...
</a>

<a class="icon-link icon-link--right">
  ...
</a>
```